### PR TITLE
Chore(Add async action creators test): Add async action creators tests

### PR DESCRIPTION
**Pull request by ogiste**
#### What does this PR do?
Add tests to cover the action creators that were previously untested for the profiles feature

#### Description of Tasks to be completed?
Write tests for asynchronous dispatch functions used in the profile view and edit feature

#### What are the relevant Pivotal Tracker Stories?
[#165973807](https://www.pivotaltracker.com/story/show/165973807)

### CODE REVIEW

File: src/actions/__tests__/resetActions.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import moxios from 'moxios';
 import thunk from 'redux-thunk';
```
Comment (Mirriam-Maina) [inline src/actions/__tests__/resetActions.js:3]: @ogiste is this Valerie's work?

Comment (ogiste) [reply src/actions/__tests__/resetActions.js:3]: Yes, I just needed to make some  adjustments to her formatting to match the eslint requirements,

**Review by Mirriam-Maina — COMMENTED** [review]
Rebase off develop

File: src/actions/__tests__/resetActions.js (hunk 1 above)
Comment (Mirriam-Maina) [reply src/actions/__tests__/resetActions.js:3]: okay

**Pull request merged**